deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -231,24 +235,28 @@ private void ingestChildren(ContentContainerObject destObj, Resource parentResc)
 	 * @throws DepositException
 	 * @throws IOException 
 	 */
-	private void ingestFileObject(ContentObject parent, Resource parentResc, Resource childResc)
+	private FileObject ingestFileObject(ContentObject parent, Resource parentResc, Resource childResc)
 			throws DepositException, IOException {
 
 		if (skipResumed(childResc)) {
-			return;
+			return null;
 		}
 
 		// TODO add ACLs
 		WorkObject work = (WorkObject) parent;
 		FileObject obj = addFileToWork(work, childResc);
-		addDescription(work);
-		// add premis events to the file object
+		// Add ingestion event for file object
+		addIngestionEventForChild(obj);
 		addPremisEvents(obj);
+		// add MODS
+		addDescription(work);
```

Comment:
I'm fairly certain this has gone back and forth between those two options at least three or four times now :-) I originally had it being added to the file object, but then we discussed it and decided the work should get it. I've changed it back for the time being.